### PR TITLE
Fix Docker

### PR DESCRIPTION
**Pull request by jeyglk**
This PR fixes the docker composition and environment, as it basically doesn't work out of the box currently.

It fixes issues https://github.com/Netflix/security_monkey/issues/637 
- Nginx crash if no SSL cert
- Database "secmonkey" does not exist

It also brings:
- Default admin user creation
- API and Scheduler wait for the DB to be up
- Docker files refactoring / cleaning
- Docker docs update

This is going beyond my previous PR https://github.com/Netflix/security_monkey/pull/651 which was fixing only the HTTP switch.

Hopefully this will make the use of this project in Docker easier for anyone who want to test it.

### CODE REVIEW

**Comment by monkeysecurity** [comment]
Fails on Windows.  I'll grab a mac or ubuntu box and try this out.

```
> docker-compose build
postgres uses an image, skipping
Building data
Traceback (most recent call last):
  File "docker-compose", line 3, in <module>
  File "compose\cli\main.py", line 64, in main
  File "compose\cli\main.py", line 116, in perform_command
  File "compose\cli\main.py", line 223, in build
  File "compose\project.py", line 313, in build
  File "compose\service.py", line 804, in build
  File "site-packages\docker\api\build.py", line 215, in build
  File "site-packages\docker\utils\decorators.py", line 47, in inner
  File "site-packages\docker\api\client.py", line 179, in _post
  File "site-packages\requests\sessions.py", line 522, in post
  File "site-packages\requests\sessions.py", line 475, in request
  File "site-packages\requests\sessions.py", line 596, in send
  File "site-packages\requests\adapters.py", line 423, in send
  File "site-packages\requests\packages\urllib3\connectionpool.py", line 595, in urlopen
  File "site-packages\requests\packages\urllib3\connectionpool.py", line 363, in _make_request
  File "httplib.py", line 1042, in request
  File "httplib.py", line 1082, in _send_request
  File "httplib.py", line 1038, in endheaders
  File "httplib.py", line 882, in _send_output
  File "httplib.py", line 844, in send
  File "site-packages\docker\transport\npipeconn.py", line 31, in connect
  File "site-packages\docker\transport\npipesocket.py", line 22, in wrapped
  File "site-packages\docker\transport\npipesocket.py", line 50, in connect
pywintypes.error: (2, 'WaitNamedPipe', 'The system cannot find the file specified.')
Failed to execute script docker-compose
```

File: docker/nginx/Dockerfile [hunk 1]
```diff
@@ -43,8 +42,7 @@ RUN /bin/rm /etc/nginx/conf.d/default.conf &&\
 WORKDIR /etc/nginx
 EXPOSE 443
```
Comment (monkeysecurity) [inline docker/nginx/Dockerfile]: Does this also need to expose 80 for non-ssl?

**Comment by monkeysecurity** [comment]
Hey @jeyglk - any idea why my containers start up but most seem to exit immediately?

I did not supply SSL certs, but I also set `SESSION_COOKIE_SECURE=False` in my secmonkey.env.

```
> docker-compose up -d
Starting secmonkey-db
Starting secmonkey-data
Starting secmonkey-api
Starting secmonkey-nginx
Starting secmonkey-scheduler

> docker-compose ps
       Name                      Command                State      Ports
--------------------------------------------------------------------------
secmonkey-api         /usr/local/src/security_mo ...   Exit 1
secmonkey-data        /bin/bash                        Exit 0
secmonkey-db          docker-entrypoint.sh postgres    Up         5432/tcp
secmonkey-nginx       /usr/local/src/security_mo ...   Exit 127
secmonkey-scheduler   /usr/local/src/security_mo ...   Exit 1
```

File: docker/nginx/Dockerfile (hunk 1 above)
Comment (jeyglk) [inline docker/nginx/Dockerfile]: Could be, but itis fine since it's exposed in the docker-compose.

**Pull request merged**